pageserver/src/layered_repository/storage_layer.rs

```diff
@@ -127,6 +129,9 @@ pub trait Layer: Send + Sync {
     /// of the *relish*, not the beginning of the segment. The requested
     /// 'blknum' must be covered by this segment.
     ///
+    /// `cached_img_lsn` should be set to a cached page image's lsn < `lsn`.
```

Comment:
Maybe it would be more convenient if cached_img_lsn was a field in PageReconstructData?

Reply:
I thought about doing this but I like that it isn't mutable when passed as an arg